ios/push-notifications.md

```diff
@@ -175,110 +225,35 @@ Make the following changes to the configuration file. The values are available i
-            UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound, .badge]) {
-                (granted, error) in
-                print("Permission granted: \(granted)")
-                // 1. Check if permission granted
-                guard granted else { return }
-                // 2. Attempt registration for remote notifications on the main thread
-                DispatchQueue.main.async {
-                    UIApplication.shared.registerForRemoteNotifications()
-                }
-            }
-        }
-
-        // Other app delegate methods...
-    }
-    ```
-
-    **Note**: If you already have push notification delegate methods, you can just add the `interceptDidRegisterForRemoteNotifications` and `interceptDidReceiveRemoteNotification` callbacks to Pinpoint client.
-
-1. In Xcode Project Navigator, choose your app name at the top, choose your app name under **Targets**, choose the **Capabilities** tab, and then turn on **Push Notifications**.
+The following steps show how to send push notifications targeted for your app.
 
-    ![Image of turning on Push Notifications capabilities in Xcode]({{image_base}}/xcode-turn-on-push-notification.png)
+1. Go to https://developer.apple.com/account/ 
 
-1. Configure the app to run in the **Release** profile instead of the default **Debug** profile. Perform the following steps to get a notification to the device:
+2. Under "Certificates, Identifiers & Profiles", on the left side click on "Keys", click +, type in a name like "Push Notification Key", check off Apple Push Notification Service (APNs). Register and download the file. It should be in the format of `AuthKey_<Key ID>.p8`
 
-    1. For your app target, go to the **General** tab of project configuration and make sure **Automatically Manage Signing** check box is not selected.
+2. Go to your Membership details page to get the TeamID
 
-    1. In the **Signing (Release)** section, choose the production provisioning profile you created on Apple developer console. For testing push notifications on a device, you will need an [Ad Hoc Provisioning Profile](https://help.apple.com/xcode/mac/current/#/dev4335bfd3d) configured with a Production AppStore and Ad Hoc certificate, and with the device(s) to be used for testing.
+3. Open the AWS Pinpoint console with `amplify console analytics`
 
-    1. In the top left corner of Xcode (where your app name is displayed next to the current build target device), choose on your app name and then select **Edit Scheme**, and then set **Build configuration** to **Release**
+4. Go to Settings, enable push notifications for APNS, select Key certificate, and add the Team ID, Key ID, the unique bundle identifier of your app, and select the `AuthKey_<Key ID>.p8` file
 
-        **Note**: Run your app on an iPhone device to test. Push notifications are not supported on simulators.
+5. Go to Campaign, provide a campaign name, and click next.
```

Comment:
Go to campaign, click Create Campaign, provide a campaign name and select Push Notifications as the channel, and click next

Reply:
updating